nuxeo-web-ui-ftest/test/features/step_definitions/support/fixtures/documents.js

```diff
@@ -89,24 +89,56 @@ fixtures.documents = {
     };
     const uploader = nuxeo.batchUpload();
     return uploader.upload(blob).then(() =>
-      nuxeo.operation('FileManager.Import')
-          .input(uploader)
-          .context(params.context)
-          .params(params)
-          .execute({ headers: { nx_es_sync: 'true' } })
-          .then((docs) => {
-            const doc = docs.entries[0];
-            liveDocuments.push(doc.uid);
-            return doc;
-          })
+        nuxeo.operation('FileManager.Import')
+            .input(uploader)
+            .context(params.context)
+            .params(params)
+            .execute({ headers: { nx_es_sync: 'true' } })
+            .then((docs) => {
+              const doc = docs.entries[0];
+              liveDocuments.push(doc.uid);
+              return doc;
+            })
     );
   },
+  addToClipboard: (document, username) => {
+    const key = `${username}-nuxeo-clipboard`;
+
+    const clipboardDocument = {
+      uid: document.uid,
+      title: document.title,
+      type: document.type,
+      path: document.path,
+      lastViewed: new Date(),
+    };
+    if (document.contextParameters && document.contextParameters.thumbnail &&
+        document.contextParameters.thumbnail.url) {
+      clipboardDocument.contextParameters = { thumbnail: { url: document.contextParameters.thumbnail.url } };
+    }
+
+    browser.execute((doc, storageKey) => {
+      const clipboard = JSON.parse(localStorage.getItem(storageKey)) || [];
+      clipboard.push(doc);
+      localStorage.setItem(storageKey, JSON.stringify(clipboard));
+    }, clipboardDocument, key);
+  },
+  clearClipboard: (username) => {
+    const key = `${username}-nuxeo-clipboard`;
+    browser.execute((storageKey) => {
+      localStorage.removeItem(storageKey);
+    }, key);
+  },
+  getDocument: (path) => nuxeo.repository().fetch(path),
```

Comment:
Because this `path` here can be either a path or an id, according to whether it starts with a `/` or not, we should name it something that points to this duality, like `ref`([which is what js client uses](https://github.com/nuxeo/nuxeo-js-client/blob/master/lib/repository.js#L47)) or `pathOrId`, for example.